log/logging.go

```diff
@@ -7,26 +7,24 @@ package log
 import (
 	"fmt"
 	"os"
-
-	"github.com/sirupsen/logrus"
-
-	"github.com/weaveworks/promrus"
 )
 
-// Setup configures a global logrus logger to output to stderr.
-// It populates the standard logrus logger as well as the global logging instance.
+// Setup configures a global Gokit logger to output to stderr.
+// It populates the Gokit logger as well as the global logging instance.
 func Setup(logLevel string) error {
```

Comment:
Can you check among the projects who use this? I can't see it being called neither in dskit, neither in Mimir.

Reply:
I have checked `dskit`, `Mimir`, `Loki`, `Tempo` and `Pyroscope` and none of them uses this method.
I will delete it.